internal/pkg/manifest/workload_ecs.go

```diff
@@ -91,17 +91,33 @@ type AlarmArgs struct {
 	MemoryUtilization *float64 `yaml:"memory_utilization"`
 }
 
+// WorkerAlarmArgs represents specs of CloudWatch alarms for Worker Service deployment rollbacks.
+type WorkerAlarmArgs struct {
+	AlarmArgs
+	MessagesDelayed *int `yaml:"messages_delayed"`
+}
+
 // DeploymentConfiguration represents the deployment strategies for a service.
 type DeploymentConfiguration struct {
 	Rolling        *string                    `yaml:"rolling"`
-	RollbackAlarms Union[[]string, AlarmArgs] // `yaml:"rollback_alarms"` 
+	RollbackAlarms Union[[]string, AlarmArgs] // `yaml:"rollback_alarms"`
```

Comment:
Just double check: now that we have `Union` we don't have to worry about the env override mergo stuff for `transform.go` at all.

Reply:
Yeah, I don't think so. I manually tested for my last PR and it...just...worked! 🪄